src/DataStructures/DataBox/ObservationBox.hpp

```diff
@@ -45,6 +48,9 @@ class ObservationBox<tmpl::list<ComputeTags...>, DataBoxType>
  public:
   /// A list of all the compute item tags
   using compute_item_tags = tmpl::list<ComputeTags...>;
+  /// A list of all the compute item tags that are compute items
+  using actual_compute_item_tags =
```

Comment:
Since the new alias was only used in one place, I just inlined it.

I think renaming to `item_tags` would be confusing as there is already a `tags_list` that means something else.